core/src/main/java/bisq/core/trade/statistics/TradeStatistics3.java

```diff
@@ -443,7 +443,8 @@ public boolean isValid() {
             if (optionalPaymentMethodById.isPresent()) {
                 maxTradeLimit = optionalPaymentMethodById.get().getMaxTradeLimitAsCoin(currency).value;
             }
-        } catch (Exception ignore) {
+        } catch (Exception e) {
+            log.warn("Error at isValid(). " + e.toString(), e);
```

Comment:
I'm prettry sure `e.toString(),  e` will result in a duplicated stack trace in the log.   Maybe `log.warn("Error at isValid(). {}", e);` is enough?

Reply:
I think so too. The first object that is passed is called `.toString` upon. I'll push a fix to this branch.

Reply:
I think even `log.warn( "Error at isValid().", e );` will be sufficient.